examples/classification/readme.md

```diff
@@ -0,0 +1,73 @@
+# HierarQML
+
+This tutorial guides on using HierarQcal to create quantum circuit models for classification tasks. It uses the modern stack of Pytorch Lightning and PennyLane for classical and quantum machine learning.
+
+## Setup
+
+Create a Python environment and install the required packages using
+
+```
+pip install -r requirements.text
+```
+
+Set the Python Path using `export PYTHONPATH=.` before executing any commands later on, or you will get errors like ModuleNotFoundError: No module named ''.
+
+## Project Structure
+
+To go through the main project structure, let's train a QCNN on GTZAN data.
+
+### Directory Structure
```

Comment:
Can we name the parent folder "classification" -> "qml_template", I will then move all examples into their own folder to make things more neat.